cms-xcache/xcache-consistency-check-wrapper.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+ENABLE=`cat /etc/environment |  grep XC_XCACHE_CONSISTENCY_CHECK | awk -F "=" '{print $2}'`
+echo "ENABLE: "$ENABLE
+
+if [ "$ENABLE" == "1" ]
+then
+    echo "Starting"
```

Comment:
```suggestion
    echo "Starting XCache consistency check..."
```

Reply:
I cannot longer commit this suggestion so I've pushed a new commit for this changed